src/Test/DockerIntegration/AcceptanceTest.php

```diff
@@ -41,11 +47,44 @@ public function testDefault()
 
         $this->assertSame(0, $code);
 
-        $process = new Process\Curl();
-        $process->setTimeout(null)
+        $assertContentPresence = function () {
+            $process = new Process\Curl();
+            $process->setTimeout(null)
+                ->run();
+
+            $this->assertSame(0, $process->getExitCode());
+            $this->assertContains('Home page', $process->getOutput());
+        };
+
+        $assertContentPresence();
+
+        $code = (new Process\Ece('config:dump', Config::CONTAINER_DEPLOY))
             ->run();
 
-        $this->assertSame(0, $process->getExitCode());
-        $this->assertContains('Home page', $process->getOutput());
+        $this->assertSame(0, $code);
+
+        $to = sys_get_temp_dir() . '/app/etc/config.php';
+
+        $read = new Process\Copy('/app/etc/config.php', $to);
+        $read->mustRun();
+
+        $config = require $to;
+
+        $arrayManager = new ArrayManager();
+        $flattenKeysConfig = implode(array_keys($arrayManager->flatten($config, '#')));
+
+        $this->assertContains('#modules', $flattenKeysConfig);
+        $this->assertContains('#scopes', $flattenKeysConfig);
```

Comment:
One last though, is it helpful to add the third parameter $message to these so that if there is a failure, it's a bit easier to troubleshoot?

Reply:
It's pretty self-explaining the kets are different to assert and will be displayed in case of fail